dynawo/sources/Models/CPP/ModelNetwork/DYNModelCurrentLimits.cpp

```diff
@@ -135,9 +135,9 @@ ModelCurrentLimits::evalZ(const string& componentName, const double& t, state_g*
 
       if (openingAuthorized_[i] && g[1 + 2 * i] == ROOT_UP) {  // Warning: openingAuthorized_ = false => no associated g
         state = ModelCurrentLimits::COMPONENT_OPEN;
-        DYNAddConstraintWithData(network, componentName, true, modelType,
-            constraintData(ConstraintData::OverloadOpen, i),
-            OverloadOpen, acceptableDurations_[i], side_);
+        if (network->hasConstraints() && !DYN::doubleIsZero(lastCurrentValue_ * factorPuToA_)) {
```

Comment:
Why modifying the way to create the constraints? 
the test of !DYN::doubleIsZero(lastCurrentValue_) should be enough